# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/PostTraversalTask.h
# Commit: aa5527055ad1
# Full Hash: aa5527055ad1d21aa109bc8a0ceaee3b0a58288c
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771493
# File Overlap Count: 1
# Description:
#   Bug 1771493 - Part 1. Split FontFace(Set) into DOM facing and implementation classes. r=emilio
#   
#   This patch aims to split the implementation details of FontFace(Set)
#   into FontFace(Set)Impl classes. The Impl variants have threadsafe
#   refcounting and are intended to be the basis for worker support for
# ==============================================================================

diff -r e0c50a17b0ff -r aa5527055ad1 layout/style/PostTraversalTask.h
--- a/layout/style/PostTraversalTask.h	Tue Jun 28 15:45:58 2022 +0000
+++ b/layout/style/PostTraversalTask.h	Tue Jun 28 15:58:09 2022 +0000
@@ -16,6 +16,7 @@
 namespace dom {
 class FontFace;
 class FontFaceSet;
+class FontFaceSetImpl;
 }  // namespace dom
 namespace fontlist {
 struct Family;
@@ -59,7 +60,7 @@
   }
 
   static PostTraversalTask DispatchFontFaceSetCheckLoadingFinishedAfterDelay(
-      dom::FontFaceSet* aFontFaceSet) {
+      dom::FontFaceSetImpl* aFontFaceSet) {
     auto task = PostTraversalTask(
         Type::DispatchFontFaceSetCheckLoadingFinishedAfterDelay);
     task.mTarget = aFontFaceSet;
@@ -101,7 +102,7 @@
     // mTarget (FontFaceSet*)
     DispatchLoadingEventAndReplaceReadyPromise,
 
-    // mTarget (FontFaceSet*)
+    // mTarget (FontFaceSetImpl*)
     DispatchFontFaceSetCheckLoadingFinishedAfterDelay,
 
     // mTarget (gfxUserFontEntry*)